rpcs3/rpcs3qt/rpcn_countries.cpp

```diff
@@ -0,0 +1,86 @@
+#include "stdafx.h"
+#include "rpcn_countries.h"
+
+namespace countries
+{
+	const std::array<country_code, 72> get_countries()
```

Comment:
This function can go and you can just make `countries` a const global variable in that namespace.